tests/bluetooth/tester/boards/frdm_rw612.overlay

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,uart-pipe = &flexcomm0;
+	};
+};
+
+&flexcomm0 {
```

Comment:
why not just set this node up at the board level? It can be left as `status = "disabled"` there, and enabled in this overlay. I ask because other tests can also benefit from a second UART node being configured, so some of these settings are likely reusable

Reply:
I agree, thanks @danieldegrasse.